content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -99,26 +99,29 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 
 If you are on Windows and using [Powershell Gallery](https://www.powershellgallery.com/) package manager, you can install and update kubectl with Powershell.
 
-To install:
-* Run the installation commands (making sure to specify a DownloadLocation):
+1. Run the installation commands (making sure to specify a `DownloadLocation`):
 
     ```
     Install-Script -Name install-kubectl -Scope CurrentUser -Force
     install-kubectl.ps1 [-DownloadLocation <path>]
     ```
     
-    {{< note >}}
-    **Note:** If you do not specify a DownloadLocation, kubectl will be installed in the user's temp Directory.
-    {{< /note >}}
-    The installer creates $HOME/.kube and instructs it to create a config file
-To update:
-* Run the update commands:
+{{< note >}}
+**Note:** If you do not specify a `DownloadLocation`, `kubectl` will be installed in the user's temp Directory.
+{{< /note >}}
+    
+The installer creates `$HOME/.kube` and instructs it to create a config file
+
+2. Test to ensure the version you installed is sufficiently up-to-date:
```

Comment:
This is listed as a 2 but is rendering as a 1.  It needs to be fixed to remove vagueness.